README.md

```diff
@@ -1,2 +1,106 @@
 # Spice
 Small shopping platform that takes customer orders and saves them into a database.
+
+# Product Order
+
+## Product Order Pseudo-code:
+
+First, I’ll create a class that will capture the order-objects attributes. When shopping in a supermarket, customers pick products of certain brands and then decide on the product weight they intend to purchase. That means that an order(object) has attributes like product, weight, and cost. One unique feature about products though is that they have fixed, attributes. 
+
+
+## Order:
+1. Product
+2. Weight
+3. Cost
+
+
+
+I’ll start out by creating an order function then class which define the object's characteristics: 
+
+```python
+def Order():
+	class Order:
+		def __init__(self,prod,weight): # order object gets to have this attributes.
+			self.prod = prod
+			self.weight = weight
+			self.cost = self.weight * 10
+
+					
+			
+```
+
+Second, I have to determine how I’ll compute the order costs and totals in order to define the catalog rendering. I have a class that molds object with `prod, weight, cost` attributes. `Order` class has a `Total` method that accumulates each product’s total cost. Next, I have to determine how the user will be able to make a choice and relay the user selections. `# In this section I've opted for using a sql database because it will be simpler to integrate the data and have an aggregated platform.`
+
+Third, I’ll display the product catalog most probably in dictionary form, parse it then allow users to make selections without creating many lines of code using `if...else` conditional statements. User input variables will be used as reference objects in the dictionary iteration and conditional statements. 
+
+```python
+import pprint
+import mysql.connector
+
+def Selection():
+
+	mydb = mysql.connector.connect(
+	host="localhost",
+	user="yourusername",
+	password="yourpassword",
+	database="Orderdb"
+			)# creating a connection between the python program and the SQL database.
+
+	mycursor = mydb.cursor()
+	mycursor.execute("CREATE TABLE orderTable (id INT AUTO_INCREMENT PRIMARY KEY, product VARCHAR(255), weight INT, cost INT)")
+
+	Beverage = {1:'Tea Masala',2:'Ginger',3:'Tea Leaves',4:'Sugar'} # catalog in dictionary data structure.
```

Comment:
I've decided I'll create a catalog in a database format, a more realistic approach when it comes to inventory management. It will be easier to parse the database using SQL queries as well as update the database during transactions.

Reply:
This will be an opportunity to practice SQL and polish. I'll create different databases that I'll work with, but first I'll have to design the databases and figure out the flow of sequence of operations.

Reply:
I think before doing all the upgrades, I'll first try to simplify the first attempt.Creating a simple database from order inputs.